distfiles.mk

```diff
@@ -144,6 +144,7 @@ DISTFILES =	.gitignore \
 	config/enerwave/zwn-sc7.xml \
 	config/enerwave/zwnrsm1plus.xml \
 	config/eurotronic/eur_cometz.xml \
+	config/eurotronic/eur_spiritz.xml \
```

Comment:
was told in my first PR that this is not wanted.
https://github.com/OpenZWave/open-zwave/pull/992#issuecomment-250114343

Reply:
Okay, will remove it again. 

I was assuming that this was specifically wanted, as that's what their [wiki](https://github.com/OpenZWave/open-zwave/wiki/Adding-Devices) implies: 

`run "make dist-update" from the top level directory if you had added any files`